builtin_op_importers.cpp

```diff
@@ -1518,6 +1528,111 @@ DEFINE_BUILTIN_OP_IMPORTER(Size) {
   return {{weights}};
 }
 
+DEFINE_BUILTIN_OP_IMPORTER(Slice) {
+  ASSERT(inputs.size() == 1, ErrorCode::kINVALID_NODE);
+  nvinfer1::ITensor* tensor_ptr = &convertToTensor(inputs.at(0), ctx);
+  nvinfer1::Dims dims = tensor_ptr->getDimensions();
+  int nbDims = dims.nbDims;
+  OnnxAttrs attrs(node);
+  auto axes = attrs.get<std::vector<int>>("axes");
```

Comment:
axes has a default value (that we probably can't support). We should explicitly check that it's present to avoid crashing here.

Reply:
Good catch, updated.

Reply:
It's really kUNSUPPORTED_NODE because it's legal for axes to not be present (we just can't support that case).